CPUTune/CPUTune.cpp

```diff
@@ -256,18 +263,18 @@ void CPUTune::disableTurboBoost()
 
 void CPUTune::disableProcHot()
 {
-    const uint64_t cur = rdmsr64(0x1FC);
+    const uint64_t cur = rdmsr64(MSR_IA32_POWER_CTL);
     
-    myLOG("disableProcHot: orig value: 0x%llx to 0x%llx", cur, cur & 0xFFFFFFFE);
-    wrmsr64(0x1FC, rdmsr64(0x1FC) & 0xFFFFFFFE);
+    myLOG("%s: change 0x%llx to 0x%llx in MSR_IA32_POWERCTL(0x%llx)", __func__,cur, cur & kDisableProcHotBit, MSR_IA32_POWER_CTL);
+    wrmsr64(MSR_IA32_POWER_CTL, cur & kDisableProcHotBit);
```

Comment:
Could we use setIfNotEqual() in this place such that kext keeps minimal touch of MSR.